.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       *common_steps
   ruby-3.0:
     docker:
-      - image: circleci/ruby:3.0
+      - image: circleci/ruby:3.0.3
```

Comment:
I don't know why the `3.0` image doesn't work, but `3.0.3` image (https://hub.docker.com/layers/circleci/ruby/3.0.3/images/sha256-89c0a90285a0b0bf4a0d1b399c421af4fd86a213d979d9d336946f2ca8a2a045?context=explore) works (https://github.com/runtastic/request_handler/pull/56/checks).